subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -9743,6 +9743,22 @@ u32_t ll_terminate_ind_send(u16_t handle, u8_t reason)
 	return 0;
 }
 
+#if defined(CONFIG_BT_CTLR_CONN_RSSI)
+u32_t ll_rssi_get(u16_t handle, u8_t *rssi)
```

Comment:
All LL return values are currently u32_t except for privacy. This will be overhauled when we address the LL "style issues" Jira.